ci-operator/jobs/openshift/cluster-dns-operator/openshift-cluster-dns-operator-master-presubmits.yaml

```diff
@@ -355,3 +355,59 @@ presubmits:
         secret:
           secretName: sentry-dsn
     trigger: (?m)^/test( | .* )unit,?($|\s.*)
+  - agent: kubernetes
+      always_run: true
```

Comment:
The indentation here looks funny—`always_run` and its siblings should be at the same indentation level as `agent`.